### DISCUSSION TITLE
Payment not working

### DISCUSSION THREAD

**Discussion by pemawangchuk2023** [community/community#106736] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have installed the Co-Pilot and I  have also used the free version before. I tried to subscribe after entering all the details of Visa card at https://github.com/github-copilot/signup & https://github.com/github-copilot/signup/billing?payment_duration=monthly, I get successful message saying that go back to VS Code and start using Co Pilot. However, at VS Code, I get message that says, "Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended."
Why subscription of co-pilot is not user friendly? Getting frustrated with Co-Pilot Subscription because charges are debited from the Visa card and while payment of its subscription is getting rejected.  OR is there any changes required at settings?

**Comment by Akash1134** [comment]
Hi @pemawangchuk2023 ,

Please sign out of your IDE and Copilot and try again. (this will re-authenticate your account)

If the issue persists, please open a ticket on our [Support page](https://support.github.com/contact) for further account related issues. 

Thanks!

**Reply by pemawangchuk2023** [reply]
While I could successfully subscribe chatgpt-40, I could not subscribe copilot. I tried several times. I think copilot is not accessible or made not accessible to our country, Bhutan.

**Comment by davecheney** [comment, marked as answer]
@pemawangchuk2023 i can see that payment was declined, hence no copilot access is available. Please contact support who will be able to assist you.

**Comment by VSYerragudi** [comment]
I have similar issue from India. On contact support CFA and selecting the appropriate question, I was redirected here. Why are payments declined by bank? Can we not get a payment gateway link that will enable better transaction processing?

**Reply by pemawangchuk2023** [reply]
There is no use. I have been trying to subscribe copilot since few months and still not working.

**Comment by ashwini-gupta1807** [comment]
I am not able to make a payment for last month.

**Comment by AtulTeamMystic** [comment]
i am not able to make the payment it always declines the payment no matter whatever card i use 
please help

**Comment by adrodriguezlopez** [comment]
I'm not able to make the payment it was working before but card expired I changed to the new provided by my bank and this was not accepted and now I don't have the Subscription to copilot pro
